src/test/compile-fail-fulldeps/proc-macro/shadow.rs

```diff
@@ -9,13 +9,12 @@
 // except according to those terms.
 
 // aux-build:derive-a.rs
-// aux-build:derive-a-2.rs
 
 #![feature(proc_macro)]
 
 #[macro_use]
 extern crate derive_a;
 #[macro_use]
-extern crate derive_a_2; //~ ERROR: cannot shadow existing derive mode `A`
+extern crate derive_a; //~ ERROR `derive_a` has already been defined
```

Comment:
Hm isn't this testing something else than before? Before it was preventing two `#[proc_macro_derive(A)]` was an error, but here it's erroring out on the `extern crate` annotations?